shoutem.push-notifications/app/middleware.js

```diff
@@ -0,0 +1,41 @@
+import _ from 'lodash';
+
+import { Alert } from 'react-native';
+
+import { SHOW_PUSH_NOTIFICATION } from './redux';
+
+import FCM from 'react-native-fcm';
+
+import { I18n } from 'shoutem.i18n';
+
+// eslint-disable-next-line no-unused-vars
+const showNotification = store => next => action => {  
```

Comment:
Preferred order of imports is:
1. rn and 3rd party
2. other extensions
3. local imports
```
import { Alert } from 'react-native';
import FCM from 'react-native-fcm';
import _ from 'lodash';
import { I18n } from 'shoutem.i18n';
import { SHOW_PUSH_NOTIFICATION } from './redux';
```

Reply:
done